code/modules/mob/living/carbon/human/species/station/traits_vr/negative.dm

```diff
@@ -96,6 +96,30 @@
 	cost = -1
 	var_changes = list("flash_mod" = 2.0)
 
+/datum/trait/hollow
+	name = "Hollow Bones/Aluminum Alloy"
+	desc = "Your bones and robot limbs are much easier to break."
+	cost = -4
+
+/datum/trait/hollow/apply(var/datum/species/S,var/mob/living/carbon/human/H)
+	..(S,H)
+	for(var/obj/item/organ/external/O in H.organs)
+		if(istype(O, /obj/item/organ/external/head))
+			O.min_broken_damage = 25 //Skulls are a bit denser
+
+		else if(istype(O, /obj/item/organ/external/chest) || istype(O, /obj/item/organ/external/groin))
+			O.min_broken_damage = 20
+
+		else if(istype(O, /obj/item/organ/external/arm) || istype(O, /obj/item/organ/external/leg))
+			O.min_broken_damage = 15
+
+		else if(istype(O, /obj/item/organ/external/hand) || istype(O, /obj/item/organ/external/foot))
+			O.min_broken_damage = 10 //About as low as it can go without misclicks shattering things
+			O.min_bruised_damage = 7
+
+		else //Whatever it is, it needs a change too
+			O.min_broken_damage = 15
```

Comment:
Bad form to assume 15 is higher than any future organs ever added. Again, use a multiplier.

Reply:
Y'mean lower, Arokha? 15 is the lowest any external organ (hands/feet specifically have this) has currently, it seems doubtful that there'll be anything worse... And even if there is, we can change that when it occurs.